launcher-core/src/main/scala/bloop/launcher/bsp/BspBridge.scala

```diff
@@ -41,11 +41,8 @@ final class BspBridge(
     bspServerStatus = None
   }
 
-  case class RunningBspConnection(bsp: BspConnection, out: ByteArrayOutputStream) {
-    def logs: List[String] = {
```

Comment:
Actually, where are these logs used? Don't we send them out immediately?

Reply:
in `BspBridge.waitForOpenBsp`

There is some logic over log message - https://github.com/scalacenter/bloop/blob/c1fd321b3f62da4fab79b19794616990c7b656c8/launcher-core/src/main/scala/bloop/launcher/bsp/BspBridge.scala#L123